### PR TITLE
Postgres docs

### PR DESCRIPTION
**Pull request by mitchdevenport**

### CODE REVIEW

**Review by sgrebnov — DISMISSED** [review]

**Review by ewgenius — DISMISSED** [review]

**Review by ewgenius — APPROVED** [review]

**Review by ewgenius — DISMISSED** [review]

File: spiceaidocs/content/en/reference/Postgres/_index.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+type: docs
+title: "Postgres"
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Postgres/_index.md]: `Postgres` -> `PostreSQL`

File: spiceaidocs/content/en/reference/Postgres/_index.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+## Dataset Source
+
+To use PostgreSQL as a dataset source, specify `postgres` as the `source` portion of `from` value for your dataset.
+
+```yaml
+datasets:
+  - from: postgres:path.to.my_dataset
+    name: my_dataset
+    acceleration:
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Postgres/_index.md]: I'd remove the acceleration node here.

File: spiceaidocs/content/en/reference/Postgres/_index.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+linkTitle: "Postgres"
+description: 'PostgreSQL reference'
+weight: 80
+---
+
+PostgreSQL can be connected to Spice AI as either a dataset source, a data store, or a data mesh.
+
+## Dataset Source
+
+To use PostgreSQL as a dataset source, specify `postgres` as the `source` portion of `from` value for your dataset.
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Postgres/_index.md]: Remove "your" language. Specify as "the" dataset (facts).

File: spiceaidocs/content/en/reference/Postgres/_index.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+linkTitle: "Postgres"
+description: 'PostgreSQL reference'
+weight: 80
+---
+
+PostgreSQL can be connected to Spice AI as either a dataset source, a data store, or for federated SQL query.
+
+## Dataset Source
+
+To use PostgreSQL as a dataset source, specify `postgres` as the `source` portion of `from` value for your dataset.
```
Comment (phillipleblanc) [inline spiceaidocs/content/en/reference/Postgres/_index.md]: `source` is an implementation detail - let's call it the selector:

> To use PostgreSQL as a dataset source, specify `postgres` as the selector in the `from` value for your dataset.

File: spiceaidocs/content/en/reference/Postgres/_index.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+```yaml
+datasets:
+  - from: postgres:path.to.my_dataset
+    name: my_dataset
+    acceleration:
+        enabled: true
+```
+
+## Data Mesh
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Postgres/_index.md]: Not sure we need a separate section for data mesh.

File: spiceaidocs/content/en/reference/Postgres/_index.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+datasets:
+  - from: postgres:path.to.my_dataset
+    name: my_dataset
+    acceleration:
+        enabled: true
+```
+
+## Federated SQL Query
+
+To use PostgreSQL for federated SQL query, specify `postgres` as the `source` portion of `from` value for your dataset, without acceleration.
```
Comment (phillipleblanc) [inline spiceaidocs/content/en/reference/Postgres/_index.md]:
```suggestion
To use PostgreSQL for federated SQL query, specify `postgres` as the selector in the `from` value for your dataset, without acceleration.
```

File: spiceaidocs/content/en/reference/Postgres/_index.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+```yaml
+datasets:
+  - from: postgres:path.to.my_dataset
+    name: my_dataset
+```
+
+## Data Store
+
+To use PostgreSQL as a data store, specify `postgres` as the `engine` for your dataset.
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Postgres/_index.md]: "as a data store for dataset acceleration"

File: spiceaidocs/content/en/reference/Postgres/_index.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+datasets:
+  - from: spiceai:path.to.my_dataset
+    name: my_dataset
+    acceleration:
+        engine: postgres
+```
+
+## Configuration
+
+You can configure the connection to your PostgreSQL by providing the following `params`:
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Postgres/_index.md]: Remove "you" and "your". E.g. The PostgreSQL connection is configured with the following parameters.

File: spiceaidocs/content/en/reference/Postgres/_index.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+
+- `pg_host`: The hostname of the PostgreSQL server.
+- `pg_port`: The port of the PostgreSQL server.
+- `pg_db`: The name of the database to connect to.
+- `pg_user`: The username to connect with.
+- `pg_pass_key`: The name of the secret containing the password to connect with.
```
Comment (phillipleblanc) [inline spiceaidocs/content/en/reference/Postgres/_index.md]:
```suggestion
- `pg_pass_key`: The name of the secret containing the password to connect with.
```

This is technically the name of the `key` within the `secret` using our nomenclature. The secret is the key-value object.

**Review by ewgenius — DISMISSED** [review]

File: spiceaidocs/content/en/reference/Postgres/_index.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
+---
+type: docs
+title: "PostreSQL"
+linkTitle: "PostreSQL"
+description: 'PostgreSQL reference'
+weight: 80
+---
+
+PostgreSQL can be connected to Spice AI as either a dataset source, a data store, or for federated SQL query.
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Postgres/_index.md]:
```suggestion
PostgreSQL can be used by the Spice runtime as a dataset source, a data store, or for federated SQL query.
```

**Review by lukekim — DISMISSED** [review]

**Review by phillipleblanc — APPROVED** [review]

**Pull request merged**